src/NodeServices/WorkspaceEvents.cs

```diff
@@ -57,11 +67,27 @@ internal static void OnWorkspaceClearing()
         /// An event that is triggered after a workspace is cleared.
         /// </summary>
         public static event WorkspaceClearedEventHandler WorkspaceCleared;
+
         internal static void OnWorkspaceCleared()
         {
             if (WorkspaceCleared != null)
                 WorkspaceCleared();
         }
+
+        public static event WorkspaceSettingsChangedEventHandler WorkspaceSettingsChanged;
+
+        /// <summary>
+        /// An event raised when workspace ScaleFactor setting is changed.
+        /// </summary>
+        /// <param name="scaleFactor"></param>
```

Comment:
I guess the XML comments here should have been intended for the public event instead of the internal method.
And it would be good if we do not introduce new empty lines in this PR. Thanks @aparajit-pratap! :smile: